services/src/main/java/org/fao/geonet/api/records/editing/InspireValidatorUtils.java

```diff
@@ -423,31 +397,33 @@ public static String isPassed(String endPoint, String testId, CloseableHttpClien
 
             response = client.execute(request);
 
-            if(response.getStatusLine().getStatusCode() == 200) {
+            if (response.getStatusLine().getStatusCode() == 200) {
 
                 ResponseHandler<String> handler = new BasicResponseHandler();
                 String body = handler.handleResponse(response);
 
                 JSONObject jsonRoot = new JSONObject(body);
 
                 try {
-                    return jsonRoot.getJSONObject("EtfItemCollection").getJSONObject("testRuns").getJSONObject("TestRun").getString("status");
+                    return jsonRoot.getJSONObject("EtfItemCollection").getJSONObject("testRuns").getJSONObject("TestRun")
+                            .getString("status");
                 } catch (JSONException e) {
                     return null;
                 }
 
-            }  else if (response.getStatusLine().getStatusCode() == 404) {
+            } else if (response.getStatusLine().getStatusCode() == 404) {
 
                 throw new NotFoundException("Test not found");
 
             } else {
-                Log.warning(Log.SERVICE, "WARNING: INSPIRE service HTTP response: " + response.getStatusLine().getStatusCode() + " for " + TestRuns_URL + "?view=progress");
+                Log.warning(Log.SERVICE, "WARNING: INSPIRE service HTTP response: " + response.getStatusLine().getStatusCode() + " for "
+                        + TestRuns_URL + "?view=progress");
             }
-        } catch(Exception e) {
+        } catch (Exception e) {
             Log.error(Log.SERVICE, "Exception in INSPIRE service: " + endPoint, e);
             throw e;
         } finally {
-            if(close) {
+            if (close) {
                 try {
```

Comment:
Check if can be used `IOUtils.closeQuietly` to improve the code readability

Reply:
Right, better